interfaces/Portal/src/app/program/program-tabs-navigation/program-tabs-navigation.component.html

```diff
@@ -1,6 +1,6 @@
 <div>
   <span
-    *ngFor="let phase of programPhases"
+    *ngFor="let phase of ProgramTabs"
```

Comment:
I'd also rename all "phase*"-things to "tab*"-things in the rest of the template below.